deploy/manifests/kip/base/statefulset.yaml

```diff
@@ -58,7 +58,7 @@ spec:
         - mkdir -p $CERT_DIR && /opt/csr/get-cert.sh
         env:
         - name: NODE_NAME
-          value: virtual-kubelet
+          value: kip
```

Comment:
I'm not seeing the default node name changed anywhere - we need to change either the default in node-cli, provide a name via a command line flag or change the option e.g. in cmd/kip/main.go.